src/lib/utilities/handle-error.ts

```diff
@@ -1,8 +1,26 @@
+import { networkError } from '$lib/stores/error-store';
 import { notifications } from '../stores/notifications';
+import type { NetworkError } from './request-from-api';
+
+export function isNetworkError(
```

Comment:
1. We can probably break this out into its own utility.
2. We can probably add some unit tests.